pennylane/qaoa/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```

Comment:
Should we also add `import qaoa` to `pennylane.__init__` so that users can do:
`import pennylane as qml` then `qml.qaoa`?

Reply:
yep

Reply:
I'm slightly confused, if a user does `import pennylane as qml`, then we should be able to do `qml.qaoa` without anything in the `__init__.py` file, right?

Reply:
As it currently is, users can do
`import pennylane.qaoa`
`from pennylane import qaoa`
But, they cannot do
`import pennylane as qml` then `qml.qaoa`, which would require changing the `__init__` of `pennylane` to make `qaoa` visible.

This is also a good thing to try yourself

Reply:
Ok, done!